chain/chain/src/types.rs

```diff
@@ -306,6 +304,21 @@ pub trait RuntimeAdapter: Send + Sync {
         }
         Ok(receipts_hashes)
     }
+
+    // TODO return Result?
```

Comment:
why would you need result here? also what's the point of this function on RuntimeAdapter at all?